### ISSUE TITLE
Enhancement: Change key press for accented characters menu

### ISSUE THREAD

**Issue by adamreusche** [tdm/android_device_fxtec_pro1#51]
As a user, I want to be able to select how accented characters appear on my screen. Instead of long pressing keys, i'd like to use an alternate long key press like CTRL+ (letter) to bring up the menu.

**Comment by enigma9o7** [comment]
adam: Do you know if you select an alternate keyboard layout, for example us-international, you get symbols in combination with the SYM key?

**Comment by adamreusche** [comment]
Hey TDM,

Gave it a shot and yes, I can see the accented characters in addition to
getting them with the long press.

More concerning, I've noticed that auto caps doesn't work. However, I
switched to the finqwerty intl us keyboard equivalent and it seems both
annoyances are gone.

Adam

**Comment by enigma9o7** [comment]
Finqwerty layouts aren't going to work as expected under lineage.  I suggest you use one of the included aosp layouts.    But I guess if it's somehow working the way you want, go ahead.

Auto caps works for me, in fact there's currently no way to disable it and it drives me crazy!   [Using third party keyboard like swiftkey does allow me to disable it however]

**Comment by adamreusche** [comment]
Yeah right? The weirdness is so odd. At least the long press issue is gone
now. I rarely use accented characters so thankfully this is a non-issue for
me.

 I'll give the aosp profiles a try too for posterity.
On an unrelated note, do you sometimes have issues with the spellchecker
not bringing up suggestions?


Adam Reusché

**Comment by adamreusche** [comment]
Using the finqwerty us the "sym" key works as it should now. I can bring up
accented characters by holding it and then pressing the key. Similar to how
the option key works on osx.

¨
Ü
´
É

You can mark this either resolved or low priority as this fixes my last
annoyance. Thanks mate.

Adam
